packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/data_sources/remote/http_clients/api_http_client.dart

```diff
@@ -11,17 +12,19 @@ class ApiHttpClient with DioMixin implements Dio {
     options = BaseOptions();
     httpClientAdapter = IOHttpClientAdapter();
     {{#enable_dev_menu}}
-    (httpClientAdapter as IOHttpClientAdapter).onHttpClientCreate = (client) {
-      if (ApiHttpClient.proxy.isNotEmpty) {
-        client.findProxy = ((uri) {
-          return 'PROXY ${ApiHttpClient.proxy}:8888';
-        });
-      }
-      if (proxy.isNotEmpty) {
-        client.badCertificateCallback = ((cert, host, port) => true);
-      }
-      return client;
-    };{{/enable_dev_menu}}
+    if (EnvironmentConfig.enableDevMenu == true) {
+      (httpClientAdapter as IOHttpClientAdapter).onHttpClientCreate = (client) {
```

Comment:
Do we still need the type cast to `IOHttpClientAdapter`? A few lines above the `httpClientAdapter` property is explicitly assigned a value of this type.

Reply:
Thats the only way that made Charles working after inserting proxy ip address. If I use only `IOHttpClientAdapter()`, Charles will not catch any requests or responses.